test/ZincAcessor.test.ts

```diff
@@ -70,21 +75,27 @@ contract("ZincAccessor", (accounts) => {
           break
         }
       }
-      const id = Identity.at(idContractAddress)
+      const id = IdentityV1.at(idContractAddress)
 
-      const userPrivileges = await id.getAccessorPurpose(userAddress)
-      assert.deepStrictEqual(userPrivileges.toNumber(), 15)
+      const userPrivileges = await id.getKeyPurpose.call(
+        web3.sha3(userAddress, { encoding: "hex" })
+      )
+      assert.deepStrictEqual(bigNumberArrayToNumber(userPrivileges), [1])
 
-      const zincPrivileges = await id.getAccessorPurpose(zinc.address)
-      assert.deepStrictEqual(zincPrivileges.toNumber(), 4)
+      const zincPrivileges = await id.getKeyPurpose.call(
+        web3.sha3(zinc.address, { encoding: "hex" })
```

Comment:
not sure why its needed but i recommend using soliditySha3 to be consistent

 https://web3js.readthedocs.io/en/1.0/web3-utils.html#soliditysha3

Reply:
i thick `call` can be omitted

Reply:
It can be omitted, but the type information works better with a call. Otherwise the type will be a transaction receipt